101-automation/azuredeploy.json

```diff
@@ -0,0 +1,207 @@
+{
+"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+"contentVersion": "1.0.0.0",
+"parameters": {
+    "workspaceName": {
+        "type": "string",
+        "metadata": {
+            "description": "Workspace name"
+        }
+    },
+    "sku": {
+        "type": "string",
+        "allowedValues": [
+            "pergb2018",
+            "Free",
+            "Standalone",
+            "PerNode",
+            "Standard",
+            "Premium"
+        ],
+        "defaultValue": "pergb2018",
+        "metadata": {
+            "description": "Pricing tier: perGB2018 or legacy tiers (Free, Standalone, PerNode, Standard or Premium), which are not available to all customers."
+        }
+    },
+    "dataRetention": {
+        "type": "int",
+        "defaultValue": 30,
+        "minValue": 7,
+        "maxValue": 730,
+        "metadata": {
+            "description": "Number of days to retain data."
+        }
+    },
+    "location": {
+        "type": "string",
+        "metadata": {
+            "description": "Specifies the location in which to create the workspace."
+        }
+    },
+	"resourcePermissions": {
+          "type": "bool",
+          "metadata": {
+            "description": "true to use resource or workspace permissions. false to require workspace permissions."
+          }
+    },
+    "automationAccountName": {
+        "type": "string",
+        "metadata": {
+            "description": "Automation account name"
+        }
+    },
+    "automationAccountLocation": {
+        "type": "string",
+        "metadata": {
+            "description": "Specify the location in which to create the Automation account."
+        }
+    },
+	"sampleGraphicalRunbookName": {
+            "type": "String",
+			"defaultValue": "AzureAutomationTutorial"
+        },
+        "sampleGraphicalRunbookDescription": {
+            "type": "String",
+			"defaultValue": " An example runbook that gets all the Resource Manager resources by using the Run As account (service principal)."
+        },
+        "sampleGraphicalRunbookContentUri": {
+            "type": "String",
+			"defaultValue": "https://eus2oaasibizamarketprod1.blob.core.windows.net/marketplace-runbooks/AzureAutomationTutorial.graphrunbook"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
This still needs to be done...

rest looks good, CI is working now...

Reply:
Completed revising the link to scripts and using the _artifactsLocation method with assistance from Tom FitzMacken.